### PR TITLE
Improve memorial quest - stele/obelisk distinction

### PR DESCRIPTION
**Pull request by mnalis**
see discussion at:
- <strike> https://github.com/streetcomplete/StreetComplete/pull/4102#pullrequestreview-1005665411 (and below) for text suggestions</strike>
- https://github.com/streetcomplete/StreetComplete/pull/4102#issuecomment-1155723066 (and below) for replacement picture

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -1022,9 +1022,9 @@ If there are no signs along the whole street which apply for the highlighted sec
     <string name="quest_memorialType_plaque">Plaque</string>
     <string name="quest_memorialType_war_memorial">War memorial</string>
     <string name="quest_memorialType_stone">Stone</string>
-    <string name="quest_memorialType_obelisk">Obelisk</string>
+    <string name="quest_memorialType_obelisk">Obelisk (pointy pillar)</string>
```
Comment (matkoniecz) [inline app/src/main/res/values/strings.xml:1025]: I still think that "pointy pillar" is not a helpful description of obelisk

Comment (mnalis) [reply app/src/main/res/values/strings.xml:1025]: As no comments on wikipedia yet, perhaps `Obelisk (usually 4-sided)`? Or some other idea?

**Comment by mnalis** [comment]
I've gimped-up the [3rd picture](https://github.com/streetcomplete/StreetComplete/pull/4102#issuecomment-1155723066) to remove red elements, as I could not find better (freely-licensed)  picture to illustrate stele.

small_18_June_1815_–_Waterloo_–_27th_(Inniskilling)_Regiment_of_Foot,_stele
small_18_June_1815_–_Waterloo_–_27th_(Inniskilling)_Regiment_of_Foot,_stele_Edited

**Comment by mnalis** [comment]
Do we have a policy how to attribute such heavily edited pictures? (I think I remember there being discussion about that, but can't seem to find it now.)

**Comment by matkoniecz** [comment]
"original image by XYZ from URL, memorial wreath removed from photo by Mnalis" should be fine

**Comment by FloEdelmann** [comment]
The image should remain square though.

**Review by FloEdelmann — APPROVED** [review]
LGTM :)

**Comment by mnalis** [comment]
I've opted out to go with just pictures for now, as there was not much buy-in for replacement texts, and text took too much space. 

Hopefully image alone will resolve main confusion between `obelisk` / `stone stele`:

Screenshot_20220619_155801_de westnordost streetcomplete mn debug

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (mnalis) [reply app/src/main/res/values/strings.xml]: I've decided to not change texts for now; they can always be changed later if it turns out it is still confusing

**Comment by matkoniecz** [comment]
Thanks for help @mnalis !

**Comment by matkoniecz** [comment]
One idea: maybe obelisk visible more from side would be a better illustration? Current one is not making clear that it is not some flat plate.

**Comment by mnalis** [comment]
> One idea: maybe obelisk visible more from side would be a better illustration? Current one is not making clear that it is not some flat plate.

Good idea. Unfortunately I'm on metered data connection right now, so can't really wade through dozens of pictures. Although if someone finds good one, I can crop/edit it and make a PR if they want; just mention me.

**Comment by matkoniecz** [comment]
Ideally it would be a small obelisk like current photo... To show that obelisk does not need to be a gigantic monument.

**Comment by mnalis** [comment]
smaller ones are also the ones that has a much better change to fit in a square, instead of high rectangle...
Not much choices at commons.wikipedia.org. I've only found [this one](https://upload.wikimedia.org/wikipedia/commons/9/9c/20030701140DR_Rheinsberg_%28Mark%29_Obelisk.jpg) which seems like a good candidate dimensions and background wise, but I'm afrait it would be not really be visible on small mobile screens :cry:; probably something with one color and higher contrast (one side in shadow?) would be needed :

20030701140DR_Rheinsberg_(Mark)_Obelisk_box 20030701140DR_Rheinsberg_(Mark)_Obelisk_box128

**Pull request merged**